src/main/java/org/asamk/signal/dbus/DbusSignalImpl.java

```diff
@@ -388,8 +389,28 @@ public String getGroupName(final byte[] groupId) {
     }
 
     @Override
-    public boolean isRegistered() {
-        return true;
+    public boolean isRegistered(String number) {
+        List<Boolean> result = isRegistered(List.of(number));
```

Comment:
Please use type inference, where possible
```suggestion
        var result = isRegistered(List.of(number));
```